hw1/02-palindrome.js

```diff
@@ -1,3 +1,51 @@
 const elem = document.querySelector('input');
+const resultPalindromeString = document.querySelector(
+  '.resultPalindromeString'
+);
+
+function handleInput(event) {
+  resultPalindromeString.classList.add('text-danger');
+  event.preventDefault();
+  resultPalindromeString.innerHTML = '';
+
+  const userInputValue = parseInt(elem.value);
+
+  if (Number.isNaN(userInputValue)) {
+    resultPalindromeString.innerHTML = '';
+    return;
+  }
+
+  if (userInputValue < 0) {
+    resultPalindromeString.innerHTML = 'Please enter a positive number.';
+    return;
+  }
+
+  const reversedValue = checkPalindrome(userInputValue);
```

Comment:
'checkPalindrome' was used before it was defined. The function should be in the code before it's used: airbnb style guide

Reply:
Fixed